### ISSUE TITLE
introduce Gratipay to Kanyi Maqubela

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#397]
http://kanyi.me/

Partner at http://www.collaborativefund.com/.

Reticketed from #384.

**Comment by whit537** [comment]
> Hello! Can you send a bit more information?

-----------

>Of course! :)
>
[Gratipay](https://gratipay.com/) is a payments company offering "payments and payroll for open work." We've processed $1M since launching 3+ years ago. Three distinctives:
>
 1. We are an [open company](http://inside.gratipay.com/big-picture/welcome), blurring the lines between users, collaborators, owners, and staff.
 1. We invented [take-what-you-want](https://gratipay.com/about/features/payroll) salaries to solve the problem of distributing revenue when those lines are blurred.
 1. We don't take a cut of payments—we are funded on our own platform via pay-what-you-want. Revenue works out to about 5% of volume.
>
We hit a legal brick wall 6 months ago, slam-shifted and relaunched as Gratipay 2.0, and are now moving about $5,000/mo for 100+ customers. Charts: https://gratipay.com/about/stats.
>
Basically we built the thing once out of balsa wood, it got knocked down, and now it's time to build it right.
>
Worth a conversation? Any of these times work for you?
>
Today at 2pm (US/Eastern)
Nov 25 at 3pm
Dec 3 at 11am

**Comment by whit537** [comment]
> Probably should've led with our mission, which is to cultivate an economy of gratitude, generosity, and love. :-)
>
http://inside.gratipay.com/big-picture/mission

**Comment by whit537** [comment]
> Maqubela said many of the entrepreneurs present at Platform Cooperativism came to him looking for funding — or already had in the past. So far, Maqubela told me over coffee, most on-demand economy apps had been designed to serve the kind of people who built them; rich, privileged knowledge workers. What he’s looking for is companies that want to build “tools for the labor class,” a group he sees as “an opportunity for a new network.” And he’s confident, from an investment perspective, that this new network is a good bet. “I think you will see a decentralized, cooperative Uber,” he said.

http://www.buzzfeed.com/carolineodonovan/meet-the-people-who-think-co-ops-are-the-way-to-build-a-bett

**Comment by whit537** [comment]
Seems to have fizzled.
